bot.py

```diff
@@ -558,8 +560,10 @@ async def sendEmbed(self, ctx, townInfo):
 class Gameplay(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-        self.currentlyActiveGames = {}
-        #self.cleanupInactiveGames.start() This should only be run if we have active games in the DB
+
+        # Start the timer if we have active games
+        if g_dbActiveGames.count_documents({}) > 0:
```

Comment:
Is there a way to do an `any()` rather than a `count()>0`? We don't care how many there are, just that there are any.